builders.py

```diff
@@ -89,11 +89,38 @@ def __init__(self, patient_position="HFS", patient_id="", patients_name = "", pa
         self.seriesbuilders = defaultdict(lambda: [])
         self.built = False
 
-    def build_ct(self, num_voxels, voxel_size, center=None, rescale_slope=1, rescale_intercept=-1024, column_direction=None, row_direction=None, slice_direction=None):
-        b = CTBuilder(self.current_study, num_voxels, voxel_size, center=center, rescale_slope=rescale_slope, rescale_intercept=rescale_intercept, column_direction=column_direction, row_direction=row_direction, slice_direction=slice_direction)
+    def build_ct(self, num_voxels, voxel_size, pixel_representation, center=None, rescale_slope=1,
+                 rescale_intercept=-1024, column_direction=None, row_direction=None, slice_direction=None):
+        b = CTBuilder(self.current_study, num_voxels, voxel_size, pixel_representation,
+                      center=center,
+                      rescale_slope=rescale_slope,
+                      rescale_intercept=rescale_intercept,
+                      column_direction=column_direction,
+                      row_direction=row_direction,
+                      slice_direction=slice_direction)
         self.seriesbuilders['CT'].append(b)
         return b
 
+    def build_mr(self, num_voxels, voxel_size, pixel_representation, center=None, column_direction=None,
+                 row_direction=None, slice_direction=None):

```

Comment:
slope/icept not applicable for MR, it was a mistake to add it and it was later removed.